public/components/layer_config/documents_config/document_layer_style.tsx

```diff
@@ -82,25 +116,15 @@ export const DocumentLayerStyle = ({ setSelectedLayerConfig, selectedLayerConfig
   };
 
   useEffect(() => {
-    if (
-      selectedLayerConfig?.style?.borderThickness < 0 ||
-      selectedLayerConfig?.style?.borderThickness > 100
-    ) {
-      setHasInvalidThickness(true);
-    } else {
-      setHasInvalidThickness(false);
-    }
+    const borderThickness = selectedLayerConfig?.style?.borderThickness;
+    const invalidThickness = borderThickness < 0 || borderThickness > 100;
+    setHasInvalidThickness(invalidThickness);
   }, [selectedLayerConfig?.style?.borderThickness]);
 
   useEffect(() => {
-    if (
-      selectedLayerConfig?.style?.markerSize < 0 ||
-      selectedLayerConfig?.style?.markerSize > 100
-    ) {
-      setHasInvalidSize(true);
-    } else {
-      setHasInvalidSize(false);
-    }
+    const markerSize = selectedLayerConfig?.style?.markerSize;
+    const invalidSize = markerSize < 0 || markerSize > 100;
```

Comment:
shall we add constant for min and max marker size?

Reply:
Yes, updated